pkg/docker/build.go

```diff
@@ -74,7 +74,10 @@ func BuildAddLabelsToImage(image string, labels map[string]string) error {
 	args = append(args, ".")
 	cmd := exec.Command("docker", args...)
 
-	dockerfile := "FROM " + image
+	dockerfile := "FROM " + image + "\n"
+	dockerfile += "COPY " + bundledSchemaFile + " .cog\n"
+	dockerfile += "RUN  /opt/venv/tools/bin/datamodel-codegen --input-file-type openapi --input " + bundledSchemaFile +
```

Comment:
I like the approach, but I don't think this needs to happen inside the model and I don't think it should be too hard for us to do it outside.

Here's my suggestion:

1. We start this process with a built cog image. 
2. We extract an OpenAPI schema from that image. 
3. We run a one-shot docker container running [`koxudaxi/datamodel-code-generator`](https://hub.docker.com/r/koxudaxi/datamodel-code-generator) to generate the python file.
4. We copy that python file back into the finalised container, and add the labels.